vllm/model_executor/layers/fused_moe/fused_moe_modular_method.py

```diff
@@ -103,11 +103,7 @@ def apply(
         expert_load_view: torch.Tensor | None = None,
         logical_to_physical_map: torch.Tensor | None = None,
         logical_replica_count: torch.Tensor | None = None,
-    ) -> torch.Tensor | tuple[torch.Tensor, torch.Tensor]:
-        # Is getattr needed?
-        zero_expert_num = getattr(layer, "zero_expert_num", 0)
-        zero_expert_type = getattr(layer, "zero_expert_type", None)
-
+    ) -> torch.Tensor:
```

Comment:
The return type can still be a tuple even without the zero experts.  This is due to the (optional) shared experts output.